recipes/OpenSMOG/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "OpenSMOG" %}
+{% set version = "0.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cbeda5f8dcadf8be64ffe6959d3102ea92c93eed14c5f2a1efc15ec76960ca96
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - CLINAME=OpenSMOG._cli:main
```

Comment:
Is this really the name of the entry point?

Reply:
yeah, I used to use it this way, there is something weird?

Reply:
CLINAME is a weird/undescriptive name for an executable in global scope. I would have expected this to be more like `opensmog=...`. If this is how it is currently setup, we can merge.